docs/search/content_type_search_reference/content_type_criteria.md

```diff
@@ -0,0 +1,22 @@
+---
+description: Content Type Search Criteria help define and fine-tune search queries for content types.
+page_type: reference
+month_change: true
+---
+
+# Content Type Search Criteria reference
+
+Content Type Search Criteria are only supported by Content Type Search (`ContentTypeService::findContentTypes`).
```

Comment:
```suggestion
Content Type Search Criteria are only supported by [Content Type Search (`ContentTypeService::findContentTypes`)](managing_content.md#finding-and-filtering-content-types).
```
I think adding a link to the main article (as you did in Sort Clauses) is a great idea 💪